src/main/java/com/github/maven_nar/NarCompileMojo.java

```diff
@@ -177,6 +177,22 @@ private void createLibrary(final Project antProject, final Library library)
       }
     }
 
+      if(getOS().equals( OS.WINDOWS ) && getArchitecture().equals("amd64"))
```

Comment:
How can we limit that ASM files will be compiled only on Specific Platform? We need to have this ability since currently we are only Compiling them on Windows 64 bit, while other Platforms- WIndows 32 bit, Linux, AIX and Solaris- We don't need them.